problems/prob1.rb

```diff
@@ -0,0 +1,10 @@
+#Given an integer array of size N, compute
+#the sum of all even numbers in this array
+
+def sum_of_even_numbers(numbers_array)
+  if numbers_array.nil? || numbers_array.empty? || numbers_array.any? { |e| e == nil  }
+    return nil
```

Comment:
You can omit `return`. Check [this](https://medium.com/rubycademy/the-return-keyword-in-ruby-df0a7f578fcb)

Reply:
I refactored all the problems with this idea. Didn't knew Ruby could do that

Reply:
You can learn a lot of things regarding ruby by using [rubocop](https://github.com/rubocop-hq/rubocop).